src/beman/optional/optional_monadic.t.cpp

```diff
@@ -1,6 +1,4 @@
-#include <beman/optional/optional.hpp>
-
-#include <beman/optional/optional.hpp>
+#include <Beman/Optional26/optional.hpp>
```

Comment:
No need for second include here, we already test this in `src/beman/optional/optional.t.cpp`.